tests/e2e/playwright-quickstart.config.js

```diff
@@ -14,7 +14,8 @@ const config = {
         trace: 'on',
         baseURL: 'http://localhost:9000/#',
         ignoreHTTPSErrors: true,
-        myItemsFolderName: "My Items"
+        myItemsFolderName: "My Items",
+        failOnConsoleError: false
```

Comment:
do we want to keep this disabled for yamcs quickstart testing? or leave it on and disable on a suite-by-suite basis as we have done with the couchDB tests?

Reply:
Let's do the latter